docs/development/ci-deployment-workflow.md

```diff
@@ -1,7 +1,5 @@
-
 # CI Deployment Workflow
 
-OpenEuropa is using [Drone][1] for continuous integration and continuous delivery.
-For more information about Drone please check [Drone](docs/development/third-party/drone.md)]
+OpenEuropa is using [Drone](https://drone.io) for continuous integration and continuous delivery.
 
-[1]: https://drone.io
+You can see a default Drone pipeline configuration in the [Drupal module template repository](https://github.com/openeuropa/drupal-module-template/blob/master/.drone.yml).
```

Comment:
This paragraph is true for a drupal module, but for a site, it's different. Can we be more specific?
Can we add a paragraph concerning the server https://drone.fpfis.eu and the connexion with github?

Reply:
I changed the link and added a small mention. We are talking about components here and not sites really.

Reply:
thanks.